### ISSUE TITLE
split by paragraph function

### ISSUE THREAD

**Issue by yonnic** [emmethalm/infiniteGPT#3]
Hello

I think it would improve the quality of the chatGPT output if the content is split by paragraphs and not by words. It will prevent the cutting off in the middle of a sentence and thus losing some meaning in the beginning and ending of each chunk

**Comment by Spudhead** [comment]
Agreed. I’d like to use this for language translation should work perfectly, not if it’s just chopping up sentences.
